spec/requests/icons_spec.rb

```diff
@@ -40,10 +40,10 @@
              :extension => "PNG")
     end
 
-    before { post "#{api}/icons", :params => params, :headers => default_headers }
+    before { post "#{api}/icons", :params => params, :headers => default_headers, :as => :form }
```

Comment:
Just for consistency's sake with I think every other spec file, separate this into 3 lines:
```
before do
  # stuff
end
```

Reply:
Done!